<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Century Gothic</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>YouGet</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/res/favicon.ico</normaloff>:/res/favicon.ico</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="central_widget">
   <widget class="QLabel" name="label_url">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Url</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_edit_urls">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>290</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="settingsBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>351</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QCheckBox" name="proxy_checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Enable the proxy</string>
     </property>
    </widget>
    <widget class="QLabel" name="file_path_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>331</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Path</string>
     </property>
    </widget>
    <widget class="QPushButton" name="set_path_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>330</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Set Path</string>
     </property>
    </widget>
    <widget class="QPushButton" name="set_proxy_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>330</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Set Proxy</string>
     </property>
    </widget>
    <widget class="QPushButton" name="check_update_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Check for Updates</string>
     </property>
    </widget>
    <widget class="QPushButton" name="about_button">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>260</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>About</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>321</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Storage path:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>231</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Proxy:</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>230</y>
       <width>361</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>110</y>
       <width>361</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>310</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>www.ingbyr.com</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="button_download">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>330</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>350</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menu_settings">
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="action_file_path"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_check_for_updates"/>
    <addaction name="action_supported_sites"/>
    <addaction name="action_report_bugs"/>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu_settings"/>
   <addaction name="menu_help"/>
  </widget>
  <action name="action_file_path">
   <property name="text">
    <string>File path</string>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="action_report_bugs">
   <property name="text">
    <string>Report bugs</string>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>About</string>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="action_check_for_updates">
   <property name="text">
    <string>Check for updates</string>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="action_supported_sites">
   <property name="text">
    <string>Supported Sites</string>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>download_btn_clicked()</slot>
 </slots>
</ui>
